doc/user-guides/authenticated-rl-for-app-developers.md

```diff
@@ -104,8 +104,11 @@ curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toy -i
 > **Note**: If the command above fails to hit the Toy Store API on your environment, try forwarding requests to the service and accessing over localhost:
 >
 > ```sh
-> kubectl port-forward -n istio-system service/istio-ingressgateway 9080:80 2>&1 >/dev/null &
-> curl -H 'Host: api.toystore.com' http://localhost:9080/toy -i
+> kubectl port-forward -n istio-system service/istio-ingressgateway-istio 9080:80 2>&1 >/dev/null &
+> export GATEWAY_URL=localhost:9080
+> ```
+> ```sh
+> curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toys -i
```

Comment:
```suggestion
> curl -H 'Host: api.toystore.com' http://$GATEWAY_URL/toy -i
```

Reply:
Oh god, we should make sure that all the guides have the same path definition of `/toys` to avoid this imo but we could do a follow up ticket for that. Will update this